failed to build kf5-kglobalaccel 5.8, require kconfig, ki18n

```
/private/tmp/kf5-kglobalaccel-DSDIbE/kglobalaccel-5.8.0$ cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kf5-kglobalaccel/5.8.0 -DCMAKE_BUILD_TYPE=None -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev
CMake Error at CMakeLists.txt:34 (find_package):
  By not providing "FindKF5Config.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "KF5Config", but CMake did not find one.

  Could not find a package configuration file provided by "KF5Config"
  (requested version 5.8.0) with any of the following names:

    KF5ConfigConfig.cmake
    kf5config-config.cmake

  Add the installation prefix of "KF5Config" to CMAKE_PREFIX_PATH or set
  "KF5Config_DIR" to a directory containing one of the above files.  If
  "KF5Config" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/tmp/kf5-kglobalaccel-DSDIbE/kglobalaccel-5.8.0/CMakeFiles/CMakeOutput.log".
See also "/tmp/kf5-kglobalaccel-DSDIbE/kglobalaccel-5.8.0/CMakeFiles/CMakeError.log".
```
```
/private/tmp/kf5-kglobalaccel-Y00SiU/kglobalaccel-5.8.0$ cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/kf5-kglobalaccel/5.8.0 -DCMAKE_BUILD_TYPE=None -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev
CMake Error at CMakeLists.txt:38 (find_package):
  By not providing "FindKF5I18n.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "KF5I18n", but
  CMake did not find one.

  Could not find a package configuration file provided by "KF5I18n"
  (requested version 5.8.0) with any of the following names:

    KF5I18nConfig.cmake
    kf5i18n-config.cmake

  Add the installation prefix of "KF5I18n" to CMAKE_PREFIX_PATH or set
  "KF5I18n_DIR" to a directory containing one of the above files.  If
  "KF5I18n" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/tmp/kf5-kglobalaccel-Y00SiU/kglobalaccel-5.8.0/CMakeFiles/CMakeOutput.log".
See also "/tmp/kf5-kglobalaccel-Y00SiU/kglobalaccel-5.8.0/CMakeFiles/CMakeError.log".
```

--- a/kf5-kglobalaccel.rb
+++ b/kf5-kglobalaccel.rb
@@ -9,9 +9,11 @@ class Kf5Kglobalaccel < Formula
 
   depends_on "cmake" => :build
   depends_on "haraldf/kf5/kf5-extra-cmake-modules" => :build
+  depends_on "haraldf/kf5/kf5-kconfig"
   depends_on "haraldf/kf5/kf5-kcoreaddons"
   depends_on "haraldf/kf5/kf5-kcrash"
   depends_on "haraldf/kf5/kf5-kdbusaddons"
+  depends_on "haraldf/kf5/kf5-ki18n"
   depends_on "qt5" => "with-d-bus"
 
   def install

